### PR TITLE
Homework 3 - Heitke

### PR DESCRIPTION
**Pull request by Razohara**
### Checklist
- [ ] Read all codes
- [ ] Answered the questions
- [x] Selected the correct branch to merge
- [x] Added Assignees
- [x] Added `homework` Label

### CODE REVIEW

File: meeting_2/what_type_is_this.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def what_type_is_this(variable):
 what_type_is_this(0 == 0 + False or True)
 what_type_is_this(0.1 + 0.1 == 0.2)
 what_type_is_this(0.1 + 0.2 == 0.3)
-what_type_is_this(1 < 3 < 4)
+what_type_is_this(1 < 3 < 4) # True is bool
```
Comment (jjiajian) [inline meeting_2/what_type_is_this.py:42]: bullshit is @Razohara

**Review by jjiajian — CHANGES_REQUESTED** [review]
looks not fine

**Review by jjiajian — APPROVED** [review]
looks good

**Review by victor77dev — APPROVED** [review]
Looks good to me. Great job!

**Pull request merged**